tremor-script/src/grammar.lalrpop

```diff
@@ -580,11 +582,20 @@ ModuleExprs: ExprsRaw<'input> = {
 
 ModuleExpr: ExprRaw<'input> = {
     Const => <>,
+    AggregateFnDecl => ExprRaw::AggregateFnDecl(<>),
```

Comment:
It's worth considering if this should perhaps live in trickle scope as tremor-script can't use aggregates it doesn't make too much sense to declare it in tremor-script